Assets/Tilemap/Tiles/Animated Tile/Scripts/AnimatedTile.cs

```diff
@@ -84,15 +84,16 @@ public override void OnInspectorGUI()
 			
 			float minSpeed = EditorGUILayout.FloatField("Minimum Speed", tile.m_MinSpeed);
 			float maxSpeed = EditorGUILayout.FloatField("Minimum Speed", tile.m_MaxSpeed);
-			if (minSpeed < 0)
+			if (minSpeed < 0.0f)
 				minSpeed = 0.0f;
-			if (maxSpeed < 0)
+				
+			if (maxSpeed < 0.0f)
 				maxSpeed = 0.0f;
+				
 			if (maxSpeed < minSpeed)
 			{
 				float temp = maxSpeed;
```

Comment:
Could remove this line, now that it is not needed.

Reply:
Sounds good